cmake/public/utils.cmake

```diff
@@ -378,6 +378,11 @@ function(torch_compile_options libname)
         /EHsc
         /bigobj>
       )
+    # Use standards conformance mode, see
+    # https://learn.microsoft.com/en-us/cpp/build/reference/permissive-standards-conformance
+    # https://learn.microsoft.com/en-us/cpp/build/reference/zc-preprocessor
+    target_compile_options_if_supported(${libname} PRIVATE "/permissive-")
```

Comment:
https://github.com/pytorch/pytorch/blob/fe285b95606e4ae60203e4b51cc18068a7336ed9/cmake/public/utils.cmake#L365 
It is existing. Even in few lines above your PR's code:
<img width="701" alt="image" src="https://github.com/user-attachments/assets/c9461187-ba47-45b1-bc87-9c0b368725de" />

Reply:
@janeyx99 could you please recall your `approve`？